New page: htmlformelement.autocomplete (#36210)
mdn/content: files/en-us/web/api/htmlformelement/autocomplete/index.md

---
title: "HTMLFormElement: autocomplete property"
short-title: autocomplete
slug: Web/API/HTMLFormElement/autocomplete
page-type: web-api-instance-property
browser-compat: api.HTMLFormElement.autocomplete
---

{{ APIRef("HTML DOM") }}

The **`autocomplete`** property of the {{DOMxRef("HTMLFormElement")}} interface indicates whether the value of the form's controls can be automatically completed by the browser. It reflects the {{htmlelement("form")}} element's [`autocomplete`](/en-US/docs/Web/HTML/Attributes/autocomplete) attribute.

## Value

> A string; the value `"on"` if the attribute is present, or `"off"` if set to `off`.

Reviewer: ```suggestion
A string; the value `"off"` if explicitly set to `"off"`, and otherwise always `"on"`.
```
